index.js

```diff
@@ -371,10 +371,15 @@ module.exports = function (config, deps) {
         .set(common.SESSION_TOKEN_HEADER, user.getUserToken())
         .end(
         function (err, res) {
+
           if (err != null) {
             return cb(err);
+          } else if (res.status !== 200) {
+            return cb(res.body);
```

Comment:
Okay, seems odd that it is designed this way, but if that is what it does then this would fix it.  Are we sure we don't return a non-200 success at any time.  For example, a 201 'Created' which wouldn't actually be an error?